change_message.js

```diff
@@ -1,7 +1,46 @@
-var regex = /https:\/\/\w+\.safelinks\.protection\.outlook\.com\/\?url=(\S+)\&amp;data[^\s\"']*/g;
+let outlook_regex = /https:\/\/\w+\.safelinks\.protection\.outlook\.com\/\?url=(\S+)\&amp;data[^\s\"']*/g;
+let proofpoint_regex = new RegExp('https://urldefense(?:\.proofpoint)?\.com/(v[0-9])/');
```

Comment:
For the sake of consistency
```suggestion
let outlook_regex = new RegExp('/https://\w+\.safelinks\.protection\.outlook\.com/\?url=(\S+)\&amp;data[^\s"\']*', "g");
let proofpoint_regex = new RegExp('https://urldefense(?:\.proofpoint)?\.com/(v[0-9])/');
```